server/lib/nl/constants.py

```diff
@@ -154,6 +154,11 @@
     'across',
 }
 
+# Replace the text in the query with this alternate text.
+QUERY_TEXT_REPLACEMENT_FOR_DETECTION: Dict[str, str] = {
+    'us states': 'states in the United States'
+}
```

Comment:
us states, states us, usa states, states usa ?

And, should we call this after stop-word replacement?

Reply:
Ah yes, true. This revealed a few more things, e.g. sometimes NER (and I noticed the Cloud API also) detect "states" on their own as a place. So also removed those.